scripts/zkai_native_attention_mlp_single_proof_gate.py

```
@@ -0,0 +1,553 @@
+        ("result", RESULT),
+        ("issue", ISSUE),
+        ("claim_boundary", CLAIM_BOUNDARY),
+    ):
+        if payload.get(key) != expected:
+            raise NativeAttentionMlpSingleProofGateError(f"{key} drift")
+    if payload.get("non_claims") != list(NON_CLAIMS):
+        raise NativeAttentionMlpSingleProofGateError("non-claims drift")
+    expected_payload = build_payload_without_mutations(context)
+    for key in ("source_artifacts", "routes", "summary", "mechanism", "validation_commands"):
+        if payload.get(key) != expected_payload[key]:
+            raise NativeAttentionMlpSingleProofGateError(f"{key} drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)